CloudCulator/Cloudculator.Core.UnitTests/CalculatorEngineUnitTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Cloudculator.Core.UnitTests
```

Comment:
Please create a separate branch following this convention:
phase1/feature/TB1-003-Implement-add-operation

Where:
 - phase1 - informs you about the development phase during which this part of code has been developed
 - feautre - informs you about new functionality
 - TB1-003 - Trello (board), Phase 1, task 003